kahuna/public/js/image/index.js

```diff
@@ -1,5 +1,7 @@
 import angular from 'angular';
 
+import {indexRoute} from '../routes/index';
```

Comment:
This isn't really the way to do routes.
My personal preference would be [to have them as a separate module](https://github.com/guardian/grid/tree/4ac0c221d20ea0143c6c4c7ae4f918b693617e59/kahuna/public/js/routes) so as not to couple them to anything.